client/a8c-for-agencies/data/sites.ts

```diff
@@ -204,7 +210,9 @@ export const mockedSites: Site[] = [
 			mobile: 46,
 			overall: 57,
 		},
-		php_version_num: 7.4,
+		php_version_num: '7.4',
```

Comment:
The php_version_num field is a number returned by the endpoint

Reply:
Reverted back to number in fb88090.